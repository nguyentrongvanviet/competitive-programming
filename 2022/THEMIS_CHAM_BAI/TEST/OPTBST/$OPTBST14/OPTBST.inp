1487
193971763 619831875 552002883 671827367 243557911 225218874 256200424 433000148 235183502 587330881 72156139 670141747 181130715 58905912 24574747 422051392 337404453 520576475 244156334 689544739 512476065 109392452 19803580 742602673 492124607 255863374 82351944 268499201 791648586 765368685 240059800 310410702 82736579 497903634 124618586 489905630 358247159 645210710 790746088 631332369 541477688 472608771 214303256 390716237 485979116 604645966 500352681 239113244 470374558 393490887 488470608 491052843 760772719 669787232 659512834 730651311 412299691 494431993 157175603 733081631 203445766 505224683 178891642 131262990 305598077 595267883 502833659 770027172 564356221 308829863 540722359 611873454 180249458 716918601 701603666 356340951 483220690 122715986 211784749 180919989 604538689 534956652 117548967 503631033 507413529 278215709 74980036 138353165 727907021 771555389 237456044 198844903 526195105 658403061 724892301 714985632 501766875 471631833 422767939 503880551 716874971 39235168 158879307 469374597 31031262 386953253 377629099 92446126 198409318 279336473 558863109 759882758 413444285 298032792 752292275 682388779 391340399 458212882 455863381 642907371 335626114 486056797 716209339 414025517 448552823 628519316 147713724 587044675 403212589 679521235 33919347 446095272 195208708 743338305 704455305 601467897 517154540 113829932 455412975 234338309 80395637 26342008 298003315 763958571 789473763 664222566 607403220 637239386 706322315 764955352 797050897 699384428 792733785 577618060 368494312 91195761 537487546 311481410 439083844 677081013 115251655 255605156 209129323 222580804 356494884 738959514 555035281 339235868 186974701 234014017 446120654 765076661 78538441 335190826 728120825 346077914 93365917 468723593 93325422 790570975 235519553 278076091 316115121 494881953 26219362 627355385 729571587 8022211 54398 145861890 752815077 776992489 625063552 616013183 102716984 681408464 498789168 718591485 648319918 748441184 607881914 502071409 363526538 10925730 647403703 648946818 557298082 407698530 343033290 745573816 108897514 23678991 375566080 768820606 252809315 510458101 93457114 461187123 453968556 707677560 118426062 496304819 495702024 17352631 627667812 371397457 776133893 445628078 380327615 12541543 756507805 184667815 508561448 304429694 46376795 733866696 480753930 476587845 15497837 391869384 297506894 105784509 562908823 49816576 315358828 393292563 671324141 94000159 159254872 775507306 179678917 617275578 29047976 129960946 257062612 343581294 671964656 126822443 521502354 586722393 186513994 660274105 410561318 516040843 671323737 213803101 459028662 250288584 62014103 201258747 557363452 436539190 580593963 184299104 294906723 176277761 107651619 506304675 358336476 341780716 431805737 200600361 770358466 423491317 188963947 166139555 142708357 430164132 421389161 631001591 522601988 302269082 577516360 512976822 160259421 687225291 516116635 547358411 559376400 745997769 777430761 295909198 733483933 376055768 375334979 468536557 531855853 300197131 579330756 99149509 227263348 138032628 440315253 565130007 297106771 24777594 195570935 630219176 482801330 304082743 794262072 720293671 790518675 632881689 174069151 520210593 235745162 370347832 653063168 651510152 7514064 91549062 261993444 21374737 341939324 571371635 642735731 49328030 389784821 522984655 318758705 750378491 357241615 399976944 776361455 296959736 394329795 708342844 784831648 104116895 24174971 53655208 157546794 10157756 677448218 93495924 691461550 357143923 797552328 494105272 142755119 785314639 82846751 212736609 403362956 529875883 576722238 86452966 550698868 516759128 409115531 82317368 151719351 575100598 88930329 506453220 228761365 3232893 435956154 9810514 602330438 550101981 343342403 9769790 747238072 447612210 294404452 226049234 690773581 303486212 135568542 429599520 755385178 21600630 24643843 616770050 411991059 435607972 351955180 213847087 330814149 254783196 564853148 698928808 765073327 374964602 399711315 710535682 457133383 253315877 218360634 634303050 99352848 276011812 334707532 126513335 638962442 32062610 629755480 495825040 222618906 638774067 528426360 122158324 485070693 413063105 107976364 705753051 37120888 800562306 163265105 707511172 175685941 180067271 555201194 670331405 584290259 735618186 44302034 144169338 471345413 320504329 392720940 135641931 420756854 97508574 659633160 545714280 778176147 484541450 517602210 700394262 166494273 419595866 549507503 17353700 681183606 436472450 775594996 62804091 786614149 243897256 171477654 476224023 370930190 536854026 735571031 396754593 531782855 443250406 573400956 670712969 322616526 538256954 51463346 681653223 122672212 337024304 435630438 593002209 635468025 97219063 323972867 290044877 248045396 276450602 494002583 76546025 670001487 790723777 21131075 749161465 72078447 223354528 652051862 234696314 229902828 229891218 733854235 47891556 648772244 767028618 371543951 669506382 289708010 60516952 562096422 460630527 63664803 197871470 561235645 184359412 429369262 144829967 438207525 481826901 439697286 587592147 101797488 372448051 697224691 361736611 544264031 533959348 348676029 756256119 676792162 253899656 108318144 597388976 77525050 126709310 517334329 371087306 433123892 574870592 455505549 366667976 375865084 556945191 630543620 778864756 555814059 485862349 18799569 539420000 678836820 425704172 410682975 800627444 539395867 601003524 388254753 236026620 512581246 128023402 283615515 232075553 322052759 236430715 387693883 357756286 725217050 797011666 127496577 429669912 394993334 347874886 521266694 500942677 41752574 373042760 751827244 790455073 80772556 478858748 81440978 122343077 283827156 205317224 712841890 554134299 143121088 80819274 708187489 794241265 624790244 428411196 619456131 65140683 172017986 104674284 546651832 46632406 459799684 348573863 398516067 401600340 557730805 354321597 612700395 665704140 256106036 743292006 748003814 51199309 297937506 309847181 738137169 15802778 265004488 698186576 425445806 631094043 406615934 45519084 586023115 523725139 725399334 736798340 509105247 578473117 215545651 240750385 340223107 201475842 684091433 676402111 264856143 692442522 570112468 503825841 772214990 785050377 261017633 366311010 658675967 6401989 250530587 467151135 457408283 254441718 437420720 257047320 490919416 70888949 151924221 749602133 546398148 272424342 185908157 610060859 387195122 696461803 73678883 363350170 77063803 263257111 44972766 259858264 517016347 70908173 616591773 127013809 139227967 248234047 471233283 496996704 796366930 646701587 77196255 467997736 688970136 205736655 775377670 339722524 172113583 783766479 249770052 509831525 441429138 310209432 725627074 368561327 140687882 373740613 285736579 497017611 48820342 433575388 478649162 50210060 396262573 672817406 751748150 393087794 145714461 138484193 58295463 540988527 684189676 318618491 563643247 583481235 486152545 637942470 794612339 635989328 294825295 572944041 93135369 750209588 544920972 258759061 414487338 4491655 643139871 96432447 641144505 255969439 228416948 327447355 486221925 157910863 357762250 699545133 788130303 22608887 160214390 135914048 35381184 197526581 706805526 653960161 574757532 423503313 346256925 704789664 742307699 273602736 768270143 72155977 715672082 523200014 98449761 247675923 362073858 781984585 314532112 159254680 586549583 115663880 499229670 655111759 574789279 436873665 444070196 721549990 523637021 574671530 624776303 457587605 162631025 297284136 395993243 532249433 177705708 481635162 664811010 223244811 748508430 170959012 626921168 289789477 744510778 284329063 355339067 646819412 685000005 25935757 291349176 794613140 790769378 713116273 277258976 464902972 766575604 538196664 296840488 456346389 208946914 275302674 483671548 599091118 320892518 532446357 112665742 305422150 178326937 144489846 669652019 183519635 690164583 593744523 309481853 113486156 100668719 732699321 187900499 607583766 77296 476521878 282373297 360824877 393472822 610465605 78375503 110998391 795786550 58130384 509147385 549449627 311615965 419828653 618132090 330676520 534174792 522939163 662252928 300634223 664537067 635866470 48477768 143912824 320820629 183386973 596867650 433601706 283426667 26331502 454935260 138031992 332214894 560441352 201884951 377640276 56248551 224562766 322996969 724033495 532618573 526602362 459448462 569289265 783236878 71534129 464146432 522723272 112058597 108358154 486725522 765596581 738739940 174078105 128712483 753174588 665277846 575383789 754441229 141290735 588444417 395507305 771724633 353144561 167608497 555953543 609983736 569974349 350377094 668318603 702345285 68118859 796200450 142940927 711323662 505447701 657318542 235300230 270758138 269469542 662425458 652994045 406676207 374569426 257173630 212345862 204719590 518337790 657892269 277732319 415639222 460155736 667221986 315866040 410140917 170469586 506389317 131240597 438122295 9742589 640990628 257190410 274836929 338156104 502925224 42705112 737912803 774940504 428662006 12660793 506415646 454228558 200637119 511068837 492076314 403911022 335147205 144785955 275313789 441021763 503639825 213293994 299608666 402248314 402199340 8800296 364526845 284398192 176641978 749759307 489824846 56522551 796447343 80629824 747142454 414237965 48616341 553805836 191796293 310231264 585859170 762282570 554935131 177961183 320529210 444853623 512084825 569242871 515051200 783540108 266092877 250816467 405119778 686271064 550080491 163827528 691957735 109096356 107085462 210712552 586063276 8944980 579581137 409462706 763430030 520872471 496516014 689648467 619979165 637978569 389761576 222055208 546588496 354725461 377013470 526909080 33289223 154746058 218161191 35541769 681080077 763240141 225437221 789917530 672913281 522530617 558242650 380683790 33390597 205708156 768247513 226443696 217455498 334598360 541251138 556841642 685974819 67891853 439338134 194506178 306668756 469825155 146000576 271797442 675381397 110460355 761605280 164603582 718426608 123761049 172731960 722551605 657712009 452686647 630617528 112282230 15917610 751691596 160515478 745389680 354694701 704258505 377975170 313215482 200034955 310698054 503884873 687494672 703781186 526041389 535534010 358485399 746115878 187324623 180783843 646496836 670496226 110421149 504641210 351256462 786857136 781998714 33612620 794694962 45762878 104044653 615099509 241127415 443604903 628934117 69420558 15037974 295163426 292879942 500749944 109518079 525959046 208881063 776731130 706259519 21013929 466892564 476073936 99689405 305675533 205500926 440938884 285813848 510694414 355575363 602800855 353638370 225691703 631036478 642310271 661122420 698628263 527985725 656338029 306807989 300062193 551761439 229689338 70811748 689168333 50162903 366290138 452103567 404751995 696383148 736273318 545407343 454851997 247964335 34046034 48410795 749547091 433397877 181008233 259297993 543486017 587362457 274919747 17800340 78079129 169255966 72182155 488979786 435166824 770369075 200907855 433856930 506873747 251245422 112959040 684022737 665000601 723570404 414127909 231870326 436970917 723393731 53503026 568249471 454307859 653443004 78202504 623069288 3888423 658615695 457799686 415628319 457438818 390065491 279302735 606969851 491181202 576708450 737858605 65101050 595732868 155772068 121663814 694131869 267347074 233909453 51795990 255738858 562815072 79058252 251775794 147942953 439038009 574414966 95483153 413222249 423018098 124071800 248010971 742701785 112707823 348874331 486986739 299736147 289382784 672204003 668828118 336727674 169414246 583387562 292836770 614099476 790304982 43366579 323519988 439580171 165067860 139267476 222985204 134873649 765971021 297024150 731410087 135328759 674744081 669050968 227021235 125185005 313947759 404286554 354736502 272826350 143507429 144806760 88991133 88360562 640017837 418288187 397417914 416413636 36803309 337158604 471989944 689321466 3843174 352355108 204353962 414286336 603828323 106949004 636388368 166353176 617551591 530089537 480805639 780014600 101398716 100987958 190847960 399268744 787726090 116073930 508735925 77122939 311459426 791948736 714652547 591636347 93888507 643538183 285970595 781905958 57304728 89299400 560483111 227867962 201936885 263224718 757656070 508570135 496173926 713336750 714738244 464599750 736895865 523645784 185754894 591906399 691288968 134494291 455262344 363886652 166923034 658429229 448870911 600561681 571900956 190594916 346259238 239716124 275071254 50289146 560903034 212746536 48971451 172118092 72268638 298577058 379081585 581114845 360969900 367348742 586027415 751562566 552191436 242715912 144008356 101843780 612337777 127151496 241116372 227298122 308490945 654500251 179628177 754077188 549912696 524231486 556981127 523791725 357533198 362716559 2814663 662283963 55494242 566243803 389980368 558216130 643685824 245056348 283391356 790430415 64005174 521103426 379190079 529213550 50726769 14958209 20617130 105096186 375662703 429395617 1530368 275845743 78444361 183695146 47302661 568896965 423044192 11600640 481453104 766318528 335437070 48809849 434172851 579718485 565364550 192802115 528886362 751279943 641941401 406071373 276323128 173987829 474041228 707522527 707074246 166027173 613437908 328584464 262495121 26644441 762738651 137258646 20216057 26041419 342595718 114433228 767805049 95966431 245252153 13929167 406649008 421770282 726032382 241202801 629150564 515656163 692593480 439554193 555092949 776131665 402500777 182426737 503744307 505368752 336605952 155851556 64332354 44821542 365052723 253439361 156840824 510419700 357855205 376088932 666788766 332088660 620461125 621428622 73208606 465312759 752761893 185295806 281304747 631905516 100031673 182414913 130012711 309642818 722522739 614199345 253744107 634152179 86113380 683992108 447657898 664348073 241956611 561564085 660537671 275598240 507666227 183098660 313642082 489715181 574216012 403039669 776282850 204355907 547951320 18030925 196024279 402230174 393974990 493500887 492630151 339905423 64487212 608079770 345434352 648202059 54492716 780610350 623152558 433456585 752826407 602080241 73254046 297337025 216291553 498050406 718757601 781774833 477177799 345203219 659728605 512540160 633829164 371271878 570131416 198969234 563227212 749103077 404812805 581530704 274041207 457937388 478874446 120490352 761294736 645947783 225882658 243380192 785178276 306867966 249824380 351753120 112708031 170608566 432559528 499722672 622815721 37918842 555001573 373623042 728970755 673972638 496167682 452681610 319015390 492465275 500705537 778765701 139276479 569274155 169505647 