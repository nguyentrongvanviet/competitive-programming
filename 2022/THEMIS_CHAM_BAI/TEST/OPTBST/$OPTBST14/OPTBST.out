5493734328507